lib/services/awsAdapter.js

```diff
@@ -46,11 +46,12 @@ function AWSAdapter (options) {
 		var groupName = process.env.AWS_DEFAULT_SECURITY_GROUP || "service-maker";
 
 		var result = {
```

Comment:
As discussed previously, `_.clone` will save you some maintenance in the future by copying all properties for you.